src/NHSD.BuyingCatalogue.Ordering.Api/Models/UpdateOrderItemResponseModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class UpdateOrderItemResponseModel
+    {
+        public IEnumerable<ErrorModel> Errors { get; set; }
```

Comment:
Would it not be better to have a generic list of errors, as wont this be done for lots of different methods

Reply:
Do you mean introduce a abstract Response class?

Reply:
I think so. I depends how many time we are going to make a class thats a list of errors (which I imagine will be a fair bit)

Reply:
Yeah. I think we keep an eye on it then & see if there is an appropriate abstraction. This could be one to bring up in the next coding standards meeting. If agreed, I will leave this for now.

Reply:
why do you need a specific response class anyway, if it's IEnumerable<ErrorModel>?

Reply:
What do you mean?